app/components/auk/abbreviated-text.hbs

```diff
@@ -0,0 +1,23 @@
+{{#if this.textTooLong}}
+  <div class="auk-o-flex">
+    {{#if (has-block)}}
+      {{yield this.abbreviatedText}}
+    {{else}}
+      {{this.abbreviatedText}}
+    {{/if}}
+    <WebComponents::AuButtonLink
+      @icon="circle-more"
+      @skin="mute"
+      @layout="icon-only"
+      class="auk-o-flex--vertical-center"
+    />
+    {{!-- TODO: This tooltip should be within the button-link, but doesn't yield when inside --}}
```

Comment:
Valt dit probleem op te lossen? het feit dat niet meer mogelijk is zorgt wel voor enkele positioneringsproblemen binnen andere schermen.

Reply:
Wel, ik weet het niet ... todo. Nu is het een positionerings-probleem, waar het daarnet een "werkt-helemaal-niet"-probleem was. @brenner-company Kan jij eens proberen de tooltip terug in de button de steken en zien of er dan html/css-gewijs iets schort (hover komt niet tot bij het juiste element misschien?)?

Reply:
Will do. Ik zet het op mijn todo lijst 👍